installer/datafiles/base_omsagent.data

```diff
@@ -209,6 +209,10 @@ if ${{PERFORMING_UPGRADE_NOT}}; then
         rmdir /etc/opt/microsoft/omsagent/$ws_id/certs/ 2> /dev/null
         rmdir /etc/opt/microsoft/omsagent/$ws_id/conf/ 2> /dev/null
         rm /opt/microsoft/omsagent/bin/omsagent-$ws_id 2> /dev/null		
+        rm /var/opt/microsoft/omsagent/$ws_id/tmp/Changetrackinginventory.xml* 2> /dev/null
+        rm /var/opt/microsoft/omsagent/$ws_id/tmp/LinuxFileChangeTracking.xml* 2> /dev/null
+        rm /var/opt/microsoft/omsagent/$ws_id/tmp/ServiceChangeTrackingInventory.xml* 2> /dev/null
+        rm /var/opt/microsoft/omsagent/tmp/*.xml 2> /dev/null
```

Comment:
This path should also include `$ws_id`. If you're looking to delete files that should only be in the primary workspace's tmp/ folder, this line should be outside of this workspace-ID loop.